### PR TITLE
Week 1 Update Content

### PR DESCRIPTION
**Pull request by rkyatham01**
I added pictures of my project and myself and put the descriptions. I edited the format and made it proper.
https://rkyatham-sps-summer22.uc.r.appspot.com/

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,12 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Rishith Kyatham Personal Portfolio</h1>
+      <p>I am a Sophomore in Computer Science double Majoring in Computer Science and Applied Mathematics & Science. My interest leans towards AI and Machine Learning and I am aiming to become an aspiring Full-Stack Developer in the future.</p>
```
Comment (acheng1) [inline portfolio/src/main/webapp/index.html]: nit: this seems like it's over 80 characters. Try to keep the lines length under 80 characters per https://developers.google.com/style/html-formatting#line-length (Apply throughout the rest of the file).

**Comment by acheng1** [comment]
I would try to follow https://google.github.io/eng-practices/review/developer/cl-descriptions.html#firstline on writing good "CL" or pull request descriptions:

The first line of a CL description should be a short summary of specifically what is being done by the CL, followed by a blank line. 

The [first line](https://google.github.io/eng-practices/review/developer/cl-descriptions.html#firstline) should be a short, focused summary, while the rest of the description should fill in the details and include any supplemental information a reader needs to understand the changelist holistically.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (rkyatham01) [reply portfolio/src/main/webapp/index.html]: Done

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,12 +8,42 @@
...
       <div id="greeting-container"></div>
     </div>
-  </body>
+    <a href="images/pic.jpg"><img src="images/pic.jpg"/></a>
+
+    <h2>Some Projects I Worked On</h2>
+    <p>GEO Data Visualizer</p>
+    <p>Developed a Web Application that prompts user for a location, 
+    contacts an external API to fetch latitude and 
+    longitude data, and visualizes it in Google Maps using Javascript.
```
Comment (acheng1) [inline portfolio/src/main/webapp/index.html]: nitnitnit: also indent here like you did for lines 13-14.

Comment (rkyatham01) [reply portfolio/src/main/webapp/index.html]: Got it

Comment (rkyatham01) [reply portfolio/src/main/webapp/index.html]: Done

Comment (acheng1) [reply portfolio/src/main/webapp/index.html]: I still don't see the indentation?

**Comment by acheng1** [comment]
> _ No description provided. _

nitnit: please add a description for the pull request per  https://google.github.io/eng-practices/review/developer/cl-descriptions.html#firstline on writing good "CL" or pull request descriptions

I would also include a link to the website so a reviewer can preview it as well....

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (rkyatham01) [reply portfolio/src/main/webapp/index.html]: Finished the edits.

**Review by acheng1 — APPROVED** [review]

**Pull request merged**